src/Game/Main.elm

```diff
@@ -20,82 +41,49 @@ type Direction
 
 type alias Player =
     { direction : Direction
-    , position : ( Int, Int )
+    , position : Point
     }
 
 
 type alias Model =
     { player : Player
-    , orbs : List ( Int, Int )
+    , orbs : List Point
+    , randomNumbers : List Int
     }
 
 
-main : Program () Model Msg
-main =
-    Browser.element
-        { init = init
-        , view = view
-        , update = update
-        , subscriptions = subscriptions
-        }
-
-
-subscriptions : Model -> Sub Msg
-subscriptions _ =
-    Sub.map KeyWasPressed Input.sub
-
-
-move : Direction -> ( Int, Int ) -> ( Int, Int )
-move dir ( x, y ) =
-    case dir of
-        Up ->
-            ( x, y + 1 )
-
-        Down ->
-            ( x, y - 1 )
-
-        Left ->
-            ( x - 1, y )
-
-        Right ->
-            ( x + 1, y )
-
+init : flags -> ( Model, Cmd Msg )
+init _ =
+    ( { player = { direction = Up, position = ( 4, 0 ) }
+      , orbs = []
+      , randomNumbers = []
+      }
+      -- ! Makes the assumption that world width and height are the same
+    , Random.generate RandomNumbers (Random.list (2 * Config.numberOfOrbs) (Random.int 0 (Config.worldWidth - 1)))
+    )
```

Comment:
This fixes the width/height thing by generating separate lists for the x and y coordinates.

We also pair them up immediately via `List.map Tuple.pair`.

```suggestion
    , Random.generate RandomNumbers <|
        Random.map2 (List.map2 Tuple.pair)
            (Random.list Config.numberOfOrbs (Random.int 0 (Config.worldWidth - 1)))
            (Random.list Config.numberOfOrbs (Random.int 0 (Config.worldHeight - 1)))
```

A couple of other changes follow from this, (so simply committing my suggestion won't work on it's own), but you'll figure it out :)